### DISCUSSION TITLE
[ShowNTell]: Lightning fast ZSH Prompt with support for Stacked Git that I have been Dog-Fooding for a year.

### DISCUSSION THREAD

**Discussion by NonLogicalDev** [stacked-git/stgit#282] [Show and tell]
I wanted to share a ZSH prompt I have been working on for a while.

* It has as close as possible to zero lag. (even in very large monorepos)
* It has full support for stacked git (which is half of the reason I made it)

Having Stacked Git state immediately visible like Git makes working with it even more of a treat. Check it out if you are curious:

https://github.com/NonLogicalDev/shell.async-goprompt

![Kapture 2022-07-26 at 10 45 33](https://user-images.githubusercontent.com/1669886/220277771-23a7b725-3d86-42c0-bb4a-ea035423efc0.gif)

**Comment by jpgrayson** [comment]
Thanks for sharing this, @NonLogicalDev.
